docs/_articles/en/user-guide/deploy-on-save.md

```diff
@@ -7,15 +7,19 @@ lang: en
 
 Whenever you save a local source file, you can enable immediate deployment of the changes to your default org. You might find this feature useful while trying various user interface and functional behavior or while trying to debug some behavior in an Apex class.
 
-To enable deploy on save, you can:
+To enable deploy on save:
 
 - Add `"salesforcedx-vscode-core.push-or-deploy-on-save.enabled": true` to the `.vscode/settings.json` file.
 - Or update Workspace settings:
   - Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
-  - Under Salesforce Feature Previews, select `Push-or-deploy-on-save: Enabled`.
-
-![Deploy on save feature](./images/deploy-on-save.png)
-
+  - Under Salesforce Feature Previews, select `Push-or-deploy-on-save: Enabled`
+  
+To trigger conflict detection on deploy on save:  
```

Comment:
@sbudhirajadoc - This isn't right.  The purpose of this setting is to always push or deploy with the 'overwrite' option of the command each time you save.  (No change to how push or deploy behave if you execute them a different way - such as command palette or a right-click menu.)  If you enable this **and** have conflict detection enabled, this will bypass that check and always overwrite.  Enabling this setting does NOT turn on conflict detection.

Reply:
How is "To bypass checking for conflicts on push or deploy when you also have conflict detection enabled:
"

Reply:
@smaddox-sf does this work for you as it is now?